compiler/record-minmax/src/Args.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Args.h"
+
+#include <iostream>
+
+namespace record_minmax
+{
+
+Args::Args(const int argc, char **argv)
+{
+  Initialize();
+  Parse(argc, argv);
+}
+
+void Args::Initialize(void)
+{
+  po::options_description desc("Allowed options");
+
+  desc.add_options()("help,h", "Print available options")(
+      "input_model,i", po::value<std::string>()->default_value(""), "Input model filename")(
```

Comment:
(optional) if we use `filename` it like without path. If this is full path, relative or absolute, then how about using `filepath` ?